ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -39,6 +39,27 @@ function set_test_provider() {
     fi
 }
 
+function mirror_release_image_for_disconnected_upgrade() {
+    # All IPv6 clusters are disconnected and
+    # release image should be mirrored for upgrades.
+    if [[ "${DS_IP_STACK}" == "v6" ]]; then
+      # shellcheck disable=SC2087
+      ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+MIRRORED_RELEASE_IMAGE=${DS_REGISTRY}/localimages/local-release-image
+DIGEST=\$(oc adm release info --registry-config ${DS_WORKING_DIR}/pull_secret.json ${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE} --output=jsonpath="{.digest}")
+oc adm release mirror --registry-config ${DS_WORKING_DIR}/pull_secret.json --from=${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE} --to=\${MIRRORED_RELEASE_IMAGE} --apply-release-image-signature
+echo "export MIRRORED_RELEASE_IMAGE=\${MIRRORED_RELEASE_IMAGE}@\${DIGEST}" >> /tmp/disconnected_mirror.conf
+EOF
+      scp "${SSHOPTS[@]}" "root@${IP}:/tmp/disconnected_mirror.conf" "${SHARED_DIR}/"
```

Comment:
Any particular reason to copy the `disconnected_mirror.conf` in the `$SHARED_DIR`? It seems to be used just here.

Reply:
Actually no particular reason. Maybe it is better just to source in here without using `$SHARED_DIR`.